ttnn/cpp/ttnn/operations/ccl/reduce_scatter/device/host/reduce_scatter_full_worker_grid.cpp

```diff
@@ -826,7 +837,27 @@ operation::ProgramWithCallbacks reduce_scatter_with_workers(
             edm_interface_addresses);
     }
 
-    // build the worker kernels
+    // build worker kernels ct
+    auto const& dummy_worker_slice = tensor_slicer.get_worker_slice(0);
+    auto worker_arg_builder = ReduceScatterWorkerArgBuilder(
+        device,
+        op_config,
+        topology_config,
+        dummy_worker_slice,
+        worker_transfer_info,
+        0,
```

Comment:
I can't tell on mobile right now but I expect the constructor need to change here and that these 0 value args are no longer needed?

Reply:
We should just delete these args now since we're passing in what would now be invalid values so the codepath that uses them would be invalid. 

That codepath (basically the constructor body) is no longer used though so we can just delete it at the same time - you'll see some macro.

Reply:
sounds good